modules/utxo-lib/src/bitgo/wallet/Psbt.ts

```diff
@@ -134,3 +158,148 @@ export function signWalletPsbt(
     psbt.signInputHD(inputIndex, signer);
   }
 }
+
+function classifyScriptType(input: PsbtInput): ScriptType2Of3 | undefined {
+  let scriptType: ScriptType2Of3 | undefined;
+  if (Buffer.isBuffer(input.redeemScript) && Buffer.isBuffer(input.witnessScript)) {
+    scriptType = 'p2shP2wsh';
```

Comment:
the convention is early returns in this lib (also my personal preference)

Reply:
It requires for inconsistency check with p2tr.